src-tauri/tauri.conf.json

```diff
@@ -30,7 +30,7 @@
   },
   "bundle": {
     "active": true,
-    "targets": ["app", "nsis"],
+    "targets": ["app", "nsis", "appimage"],
```

Comment:
Does AppImage support auto-update?

Reply:
I have no experience with Tauri, but the documentation seems to indicate that it supports updating with Linux and Approle: [https://v2.tauri.app/plugin/updater/](https://v2.tauri.app/plugin/updater/)

Reply:
Can you give me an AppImage build please

Reply:
I've just published a release with the Linux version on my repo.
[https://github.com/alviss7/ganymede-app/releases/tag/1.6.0](https://github.com/alviss7/ganymede-app/releases/tag/1.6.0)

Reply:
Thanks @alviss7, why does the AppImage is 152 MB? Is there no compression? Does it includes the Webview dependencies?

Reply:
Yes it's really big, I was surprised at first, and looking closer this is unfortunately normal 

Appimage integrates all dependencies including webkit (the Webview)

See: https://v1.tauri.app/fr/v1/guides/building/linux/#appimage


If you want to have a multi-distribution package and not integrate webkit, I think you should look at flatpak (with flathub). I think it would be a good idea to look into this (I can do it), but in any case it can be done in addition to appimage.

Reply:
Make more sense now. We need something that support auto-update, and Tauri seems to only support AppImage distribution

Reply:
Yes but this will changes the way how we force the user to update the app today